File: Python-Programing-Basics/First Steps in Coding/First Steps in Coding - Lab/09_yard_greening/yard_greening.py
yard_sq_m = float(input())


def calculation_expenses(yard):
    price_yard = yard * 7.61
    discount = price_yard * 0.18
    print(f"The final price is: {price_yard - discount} lv.")
    print(f"The discount is: {discount} lv.")


calculation_expenses(yard_sq_m)

File: JavaScript/Programming Basics - JavaScript/First Steps In Coding/yardGreening.js
function yardGreening(input){
    let area = Number(input[0]);
    let price = area * 7.61;
    let discount = price * 0.18;
    let totalPrice = price - discount;
    console.log(`The final price is: ${totalPrice} lv.`);
    console.log(`The discount is: ${discount} lv.`);
}
yardGreening(['550']);